features/create.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Create a post
+
+As an author
+I want to create a post
+So that I can share my knowledge with the world
+
+Scenario: Creating a valid post
+Given I am on the post submission page
+When I create a valid post
+Then I should be able to enter the post title
```

Comment:
I don't think "enter" is the word you mean for this or the following two steps. You probably wanted "view".